website/src/pages/HomePage/components/HomePageHero.tsx

```diff
@@ -0,0 +1,125 @@
+/*
+Copyright 2024 The Vitess Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { Link } from "react-router-dom";
+import Icon from "@/common/Icon";
+import { Button } from "@/components/ui/button";
+import DailySummary from "@/common/DailySummary";
+import useDailySummaryData from "@/hooks/useDailySummaryData";
+import { useState } from "react";
+import { Skeleton } from "@/components/ui/skeleton"
+
+export default function HomePageHero() {
+  const [benchmarkType, setBenchmarktype] = useState<string>("");
+
+  const {
+    dataDailySummary,
+    isLoadingDailySummary
+  } = useDailySummaryData(['OLTP', 'TPCC']);
+
+  return (
+    <section className="flex flex-col items-center h-screen p-page">
+      <h1 className="text-3xl md:text-6xl font-semibold text-center mt-10 leading-normal">
+        Benchmarking <br />
+        System for <br />
+        <span className="text-orange-500"> Vitess</span>
+      </h1>
```

Comment:
Let's add a bit more space between the lines just to match the Figma board:

**Localhost**
<img width="490" alt="image" src="https://github.com/vitessio/arewefastyet/assets/35779988/138ee81b-b37c-4511-bcdc-c5d4a9221f71">

**Figma**
<img width="490" alt="image" src="https://github.com/vitessio/arewefastyet/assets/35779988/da9466f4-b41e-4515-8f6f-28672383c544">